site/source/user-manual/ssh.rst

```diff
@@ -97,42 +97,24 @@ Community member `BrewsBitcoin <https://brewsbitcoin.com>`_ has created `a guide
 Using SSH Over Tor
 ------------------
 
-.. note:: The following guide requires that you have already added an :ref:`SSH key to your Embassy<ssh>`.
+.. note:: The following guide requires that you have already added an :ref:`SSH key to your Start9 server<ssh>`.
 
-.. caution:: SSH over Tor is only supported on Linux, though it may also work on Windows with `Torifier <https://torifier.com/>`_.
```

Comment:
Not sure what torifier.com was but it seems to no longer exist, and isn't necessary as SOCKS5 support is built into PuTTY.